### PR TITLE
Split up persistence documentation

### PR DESCRIPTION
**Pull request by chbatey**
Into:
* User facing persistence
* Building a Journal
* Persistent FSM so new users don't see it as easily (also add a warning
it will be replaced by typed persistence)

### CODE REVIEW

**Comment by ktoso** [comment]
This is only a split or should I also review for changes?

**Review by ktoso — APPROVED** [review]
Split LGTM, if I should review in depth though let me know

**Comment by chbatey** [comment]
The bit at the end of the persistence-journal doc is new

**Comment by chbatey** [comment]
As well as the warning not to use persistent FSM as it'll be replaced by typed persistence

File: akka-docs/src/main/paradox/persistence.md
Comment (ktoso) [inline akka-docs/src/main/paradox/persistence.md]: I would suggest avoiding the trap of "see here" links, they're not so well discoverable rather than "See an introduction to event sourcing on microsoft ..." but I can live with it I guess..

File: akka-docs/src/main/paradox/persistence-fsm.md
Comment (ktoso) [inline akka-docs/src/main/paradox/persistence-fsm.md]: good thanks

**Comment by ktoso** [comment]
Thanks, had a small comment then, LGTM otherwise.
Address and/or merge at will

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**
